tests/foreman/api/test_contentviewfilter.py

```diff
@@ -346,11 +353,12 @@ def test_update_cvf_with_new_name(self, new_name):
         cvf = entities.RPMContentViewFilter(
             content_view=self.content_view,
         ).create()
-        cvf.name = new_name
-        self.assertEqual(cvf.update(['name']).name, new_name)
+        for name in _positive_data():
+            with self.subTest(name):
+                cvf.name = name
+                self.assertEqual(cvf.update(['name']).name, name)
```

Comment:
I've changed the format of all of the `test_update_*` tests. Formerly, the tests had this structure:

    for name in names:
        create a content view filter
        update the content view filter
        make assertions about the updated content view filter

Now, the tests have this structure:

    create a content view filter
    for name in names:
        update content view filter
        make assertions about updated content view filter

As a result, the update_* tests will now run much more quickly. More gains could be made by defining a class specifically for the update tests:

```python
class ContentViewFilterUpdateTestCase(APITestCase):
    # set some class attribute to disable multi-threading

    @classmethod
    def setUpClass(cls):
        cls.content_view = ...
```

So only one content view filter would need to be made for the entire collection of update tests. But I haven't made that change here.